xml/templates/civicrm_state_province.tpl

```diff
@@ -2386,19 +2386,18 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (3947, 1161, "06", "Buskerud"),
 (3948, 1161, "20", "Finnmark"),
 (3949, 1161, "04", "Hedmark"),
-(3950, 1161, "12", "Hordaland"),
 (3951, 1161, "15", "Møre og Romsdal"),
 (3952, 1161, "18", "Nordland"),
 (3953, 1161, "17", "Nord-Trøndelag"),
 (3954, 1161, "05", "Oppland"),
 (3955, 1161, "03", "Oslo"),
 (3956, 1161, "11", "Rogaland"),
-(3957, 1161, "14", "Sogn og Fjordane"),
-(3958, 1161, "16", "Sør-Trøndelag"),
-(3959, 1161, "06", "Telemark"),
-(3960, 1161, "19", "Troms"),
-(3961, 1161, "10", "Vest-Agder"),
-(3962, 1161, "07", "Vestfold"),
+(3957, 1161, "16", "Sør-Trøndelag"),
+(3958, 1161, "06", "Telemark"),
+(3959, 1161, "19", "Troms"),
+(3960, 1161, "10", "Vest-Agder"),
+(3961, 1161, "07", "Vestfold"),
+(3962. 1161, "46", "Vestland")
```

Comment:
@midtsveen this punctuation will cause a fatal syntax error

Reply:
Updated